brake/backends/cachebe.py

```diff
@@ -71,7 +73,7 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            if counters[counter][0] > count:
```

Comment:
I'm not very clear on what you're wary of, I'm afraid. This is always going to have a value, and it's going to be the value that was set for the cache earlier on. If it's not, the way it was before would break as well. I'm not seeing why a tuple is more "dangerous" than an int.